### PR TITLE
Remove unhelpful warning and dependency on log crate

### PR DESCRIPTION
**Pull request by dabreegster**
When you call `to_geojson` on a large input with no feature IDs set, you currently see `Ignoring feature id property because of invalid type` spammed to logs. This doesn't seem like the most helpful or actionable warning to me, so this PR removes it entirely.

Alternatively, maybe it'd be more appropriate to show this warning only when feature ID is set to `Some(x)` but not a string or number?

### CODE REVIEW

File: src/to_geojson.rs [hunk 1]
```diff
@@ -63,10 +61,7 @@ fn make_feature_id(id: Option<JsonValue>) -> Option<FeatureId> {
     match id {
         Some(JsonValue::Number(i)) => Some(FeatureId::Number(i)),
         Some(JsonValue::String(i)) => Some(FeatureId::String(i)),
-        _ => {
```
Comment (dabreegster) [inline src/to_geojson.rs:66]: The alternate proposal: `Some(_) => log::warn` and `None => None` without any logging

**Review by lnicola — APPROVED** [review]

**Review by lnicola — APPROVED** [review]

**Comment by dabreegster** [comment]
Rebased. OK to merge?

**Comment by lnicola** [comment]
Um, who's the maintainer of this crate these days?

**Comment by dabreegster** [comment]
Hmm, you approved + merged the last few PRs. Just need to invoke bors, if the change looks OK?

**Comment by lnicola** [comment]
There's no more bors.

bors r+

**Pull request merged**